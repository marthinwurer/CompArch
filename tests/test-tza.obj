20 1 00000
21 1 00001
22 1 00005
23 1 fffff
24 1 ffffb
25 1 00bad
100 1 10020
101 1 32103
102 1 10025
103 1 10021
104 1 3210a
105 1 30107
106 1 10025
107 1 10022
108 1 3210a
109 1 3010b
10a 1 10025
10b 1 10023
10c 1 3210e
10d 1 3010f
10e 1 10025
10f 1 10020
110 1 32112
111 1 10025
112 1 10024
113 1 32115
114 1 30116
115 1 10025
116 1 00000
100
