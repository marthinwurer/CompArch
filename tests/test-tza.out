CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to 100
100:  10020  LDA   020  A[00000]  B[00000]  XR[000]
101:  32103  TZA   103  A[00000]  B[00000]  XR[000]
103:  10021  LDA   021  A[00000]  B[00000]  XR[000]
104:  3210a  TZA   10a  A[00001]  B[00000]  XR[000]
105:  30107  TR    107  A[00001]  B[00000]  XR[000]
107:  10022  LDA   022  A[00001]  B[00000]  XR[000]
108:  3210a  TZA   10a  A[00005]  B[00000]  XR[000]
109:  3010b  TR    10b  A[00005]  B[00000]  XR[000]
10b:  10023  LDA   023  A[00005]  B[00000]  XR[000]
10c:  3210e  TZA   10e  A[fffff]  B[00000]  XR[000]
10d:  3010f  TR    10f  A[fffff]  B[00000]  XR[000]
10f:  10020  LDA   020  A[fffff]  B[00000]  XR[000]
110:  32112  TZA   112  A[00000]  B[00000]  XR[000]
112:  10024  LDA   024  A[00000]  B[00000]  XR[000]
113:  32115  TZA   115  A[ffffb]  B[00000]  XR[000]
114:  30116  TR    116  A[ffffb]  B[00000]  XR[000]
116:  00000  HALT       A[ffffb]  B[00000]  XR[000]

Machine Halted - HALT instruction executed

Simulated time 90 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
